ferretdb/ferretdb_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+// ExampleRun is a testable example for Run func.
+func ExampleRun() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	conf := Config{ConnectionString: "postgres://postgres@127.0.0.1:5432/ferretdb"}
+
+	Run(ctx, conf)
+	cancel()
+	fmt.Println(GetConnectionString())
+	// Output: postgres://postgres@127.0.0.1:5432/ferretdb
```

Comment:
That function (or probably a method) should return MongoDB URI for Go driver to use

Reply:
> That function (or probably a method) should return MongoDB URI for Go driver to use

It doesn't matter, isn't it? That's a test: what will be passed that value will be.

Reply:
> There is a difference between PostgreSQL connection string and MongoDB connection string. Our users would like to connect to FerretDB as if it is was MongoDB, not connect to PostgreSQL

done.

Reply:
> There is a difference between PostgreSQL connection string and MongoDB connection string. Our users would like to connect to FerretDB as if it is was MongoDB, not connect to PostgreSQL

changed.